src/main/drivers/adc_stm32h7xx.c

```diff
@@ -315,24 +321,30 @@ void adcInit(const adcConfig_t *config)
             // Found a tag map entry for this input pin
             // Find an ADC device that can handle this input pin
 
-            for (dev = 0; dev < ADCDEV_COUNT; dev++) {
-                if (!adcDevice[dev].ADCx
+            bool useConfiguredDevice = (dev != ADCINVALID) && (adcTagMap[map].devices & (1 << dev)); {
```

Comment:
Does this ```{``` belong to the ```if``` below perhaps?

Reply:
Oops. That's an interesting one. Well spotted. :smile: